pkg/didcomm/crypto/legacy/authcrypt/encrypt.go

```diff
@@ -60,7 +43,12 @@ func (c *Crypter) Encrypt(payload []byte, sender cryptoutil.KeyPair, recipientPu
 
 	var recipients []recipient
 
-	recipients, err = c.buildRecipients(cek, edSender, recipientEdKeys)
+	var recKeys []string
+	for _, key := range recipientPubKeys {
```

Comment:
use capacity here e.g ``recKeys := make( []string,0,len(recipientPubKeys))

Reply:
I can set its length and set elements instead of appending.
I'll do the same in `buildRecipients` for the slice it constructs.